dashboard/config/environments/adhoc.rb

```diff
@@ -15,7 +15,7 @@
 
   # Consider disabling full error reports for profiling/load-testing by setting
   # this flag to false, due to memory leak: https://github.com/rails/rails/issues/27273
-  config.consider_all_requests_local = ENV['STACK_NAME'] != 'bugcrowd'
+  config.consider_all_requests_local = ENV['STACK_NAME'].to_s.exclude?('bugcrowd')
```

Comment:
I don't think we want to apply this logic to _all_ adhocs with `bugcrowd` in the name; for example, an adhoc created off of this branch.

Any reason not to do:

```suggestion
  config.consider_all_requests_local = ENV['STACK_NAME'] != 'adhoc-bugcrowd'
```

(both here and in `cloud_formation_stack.yml.erb` above)

Reply:
Agreed. I don't know why I was just doing a partial string match anyways.  It's updated now.